site/content/docs/5.2/components/buttons.md

```diff
@@ -298,6 +312,13 @@ The button plugin allows you to create simple on/off toggle buttons.
 Visually, these toggle buttons are identical to the [checkbox toggle buttons]({{< docsref "/forms/checks-radios#checkbox-toggle-buttons" >}}). However, they are conveyed differently by assistive technologies: the checkbox toggles will be announced by screen readers as "checked"/"not checked" (since, despite their appearance, they are fundamentally still checkboxes), whereas these toggle buttons will be announced as "button"/"button pressed". The choice between these two approaches will depend on the type of toggle you are creating, and whether or not the toggle will make sense to users when announced as a checkbox or as an actual button.
 {{< /callout >}}
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant with only one toggle button can not be used because it does not respect the Orange Design System specifications.
+
+**Watch out**: The component named "toggle buttons" in Boosted is not the same component as the one in ODS which behaves like a radio button.
```

Comment:
You would add a link towards the `Toggle buttons` page https://system.design.orange.com/0c1af118d/p/59c349-toggle-buttons/b/91bf23 on the DSM ?
Yes, why not...

I know it be be strange to mention these differences, but at the same time, the purpose of these callouts  is to guide the user to avoid misunderstanding in the usage of Boosted material and the Orange design system...

Reply:
Instead of the "watch out: ...", say: design not good + can't be alone + from a usability point of view use refer to looks like a checkbox => redirect towards checkbox, radio button or radio toggle button

=> Add a callout msg in docs/5.2/forms/checks-radios/#checkbox-toggle-buttons